parallel-consumer-python/pyallel_consumer/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
add parent like other modules?

Reply:
This is a static file used independently of the other files
It is shipped with the python package and on import of `pyallel_consumer` the JRE is installed, followed by using maven to install these dependencies by using this static pom file
See the `_Maven` class in `__init__.py`
```python
maven = _Maven()
maven.install_dependencies()
```

So I can either include the other pom files, but then maven may try to do too much?
Or I can dynamically load some of these properties in `setup.py` when creating the python package